### PR TITLE
Make lastline.sh ignore empty files

### PR DESCRIPTION
**Pull request by kugelfuhr**
This fixes issue  #2514.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,4 +9,4 @@
 /cc65.zip
 /util/atari/*.exe
 /util/gamate/*.exe
-
+targettest/cbm/cbmread.prg
```
Comment (mrdudz) [inline .gitignore:12]: would you remove this from the PR please?

Comment (kugelfuhr) [reply .gitignore:12]: Why? What's wrong with it?

Comment (mrdudz) [reply .gitignore:12]: Well. IF we start adding the test binaries to the git ignores... then all of them, using proper wildcards. a single one is just weird :) (and i prefer different things in separate commits)

Comment (kugelfuhr) [reply .gitignore:12]: Sounds a bit like bike shedding looking at the current file contents, but ok.

**Comment by mrdudz** [comment]
thanks!

**Pull request merged**